spec/requests/api/article_index_spec.rb

```diff
@@ -0,0 +1,19 @@
+RSpec.describe "GET /api/articles", type: :request do
+  let(:journalist) { create(:user, role: "journalist") }
+  let(:journalist_headers) { journalist.create_new_auth_token }
+  let!(:articles){5.times{create(:article, author_id: journalist.id)}}
+
+  describe "GET /api/articles" do
+    before do
+      get "/api/articles"
+    end
+
+    it "is expected to return 200 status" do
+      expect(response).to have_http_status 200
+    end
+
+    it "is expected to return a collection of articles" do
+      expect(response_json.count).to eq 5
+    end
+  end
```

Comment:
please extend the tests where you check that you get all the information you want from each article, the title, content and so on

Reply:
we have now added code in the test to test for author, title, subtitle and content